Makefile

```diff
@@ -46,7 +46,7 @@ $(OUTDIR)/%.o: $(SRCDIR)/%.cc $(HEADERS)
 .PHONY : clean
 clean:
 	@rm -f $(OUTDIR)/*.o
-	@rm -f $(OUTDIR)/$(TARGET)Dict.*
```

Comment:
Can't we still fix it (I think it's another "mistake" change from a previous PR)? At least it would be right in the latest version where it exists :-) (as of now the "make clean" removes something that doesn't exist). Anyhow I'm ok to deprecate it soon.

Reply:
ok